.github/workflows/build_ci_image.yml

```diff
@@ -68,10 +70,13 @@ jobs:
         uses: docker/build-push-action@v6
         with:
           context: .
+          cache-from: type=gha
+          cache-to: type=gha,mode=max`
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Congratulations @twsearle 👍🏽 
Just being picky really, will it be better to have some sensible tag and label for the image? like <alma9>:<version>

Reply:
The tags and labels are automatically generated from the [docker metadata action](https://github.com/docker/metadata-action). Its all still annoyingly chicken and egg! But I am hoping after I merge this I can version the tags the next PR.